targets/riscv.json

```diff
@@ -1,26 +1,12 @@
 {
```

Comment:
We may need to split our target specs into 32 and 64 bit RISC-V? I think that we also probably need to use a build tag to distinguish between the two.

Reply:
I agree, now there are two files and I also added a new `tinygo.riscv64` build tag. However I have kept the `tinygo.riscv` build tag for 32-bit architectures for backward compatibility. I don't know if it would be better to replace it by `tinygo.riscv32` to be more explicit?